Kempe/Kempe.txt

```diff
@@ -0,0 +1,145 @@
+
+This code is contributed by Riyazul555
+
+
+Time Complexity = O(N^2)
+Space Complexity = O(N)
+
+
+This is the PSEUDOCODE of Kempe's Algrithm 
+
+
+
+
+procedure BFS(nod, G, viz)
```

Comment:
Please use the syntax of functions that is used in other pseudocode implementations in this repository ([example](https://github.com/iuliagroza/Algorithms/blob/1f516a04fa02c2c5c7277e2f3b350994cca70386/Dijkstra/Dijkstra.txt#L8)). Fix everywhere where applicable.